driving-side/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+use geo::{Contains, Point};
+use geojson::{quick_collection, GeoJson};
+
+/// Returns true if the point is located in a country with left-handed driving.
+pub fn is_left_handed(pt: Point) -> bool {
+    let raw = include_str!("../data.geojson");
```

Comment:
Every single query does the work of parsing the geojson -- surely that's slow? In practice, not at all -- `cargo test` in debug mode is under 0.01s for the 4 cases. And the usage pattern of this API is once per map import, so the time is absolutely negligible.

We could optimize later by
1) making an API to do the loading+parsing once, then query many times
2) using a quadtree like https://github.com/georust/rstar to prune the lookup
3) Serializing the quadtree and `geo::Polygon`s directly in a binary format and deserializing that instead

Reply:
Oh yeah, and including this crate adds about 80KB to build size from `data.geojson`. That's also negligible and could be made smaller by step 3 above